doc/devguide/contributing.rst

```diff
@@ -0,0 +1,110 @@
+Contributing
+============
+
+Thanks for Your Help!
+---------------------
```

Comment:
Is there a reason the information from the `CONTRIBUTING.md` file is repeated here?

Reply:
So this is what ends up on the webpage separate from the repo?

Reply:
Correct

Reply:
I used pandoc to generate the .rst file from the .md file